monty_hall_function.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+from enum import Enum
+
+class Strategie(Enum):
+    CHANGER = 1
+    GARDER = 2
+    
+def Monty_Hall(nb_de_parties = 10000):
+    
+    '''Simule n parties du jeu de Monty Hall
+    
+    Cette fonction simule n = nb_de_parties du jeu
+    de Monty Hall pour chaque strategie "CHANGER" et
+    "GARDER". Elle retourne les gains de chaque
+    strategie
+    
+    args :
+        nb_de_parties (int) : Nombre de parties
+        de chaque strategie
+        
+    return : 
+        np.array (2,) : Gains de chaque methodes, 
+        d'abord "CHANGER" puis "GARDER"
+    '''
+    #Bonne porte au hasard entre 0 et 2 pour toutes les parties
```

Comment:
Always try to put a white space before the commented text:
`# Bonne porte ...` instead of `#Bonne porte ...`